src/sim/ticked.cc

```diff
@@ -35,9 +35,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "sim/ticked_object.hh"
+#include "sim/ticked.hh"
 
-#include "params/TickedObject.hh"
+#include "params/Ticked.hh"
```

Comment:
This doesn't seem right to me. I think this is still TickedObject since that's the name of the SimObject in Ticked.py